ChangeLog.md

```diff
@@ -1,6 +1,17 @@
 
 # Change Log
 
+## Version 10.25.1
+
+### Security fixes
+
+1. Updated Golang to 1.22.4 to address security vulnerabilities
```

Comment:
Should we include this too in the release? https://github.com/Azure/azure-storage-azcopy/pull/2713

Reply:
@gapra-msft would the Go SDK have to release first?  Or is that not how it works in Go.

Reply:
its been released already https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/azidentity/CHANGELOG.md#160-2024-06-10

Reply:
We dont have to wait for azblob to upgrade I dont think

Reply:
I guess this isn't a blocking step to release, so we can ignore for now and plan to have this in the next release.

Reply:
@gapra-msft, we don't have to wait for AzBlob.  We just need CI to finish on https://github.com/Azure/azure-storage-azcopy/pull/2713 and we can include it in the release.